UKHomeOffice/cop-react-components
# COP-10819: Fix font rendering

### Description
Fixed the font rendering in Storybook. The weights were incorrect as it was trying to embolden a regular font weight, which doesn't work properly.

https://support.cop.homeoffice.gov.uk/browse/COP-10819

### To test
Run it locally with `yarn storybook` or look at https://ukhomeoffice.github.io/cop-react-components to see it working.
